extensions/smallrye-stork/deployment/src/test/java/io/quarkus/stork/HealthExtensionCheckTest.java

```diff
@@ -0,0 +1,44 @@
+package io.quarkus.stork;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.LogRecord;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.DisabledOnOs;
+import org.junit.jupiter.api.condition.OS;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.builder.Version;
+import io.quarkus.maven.dependency.Dependency;
+import io.quarkus.test.QuarkusUnitTest;
+import io.quarkus.test.common.QuarkusTestResource;
+
+@DisabledOnOs(OS.WINDOWS)
+@QuarkusTestResource(ConsulContainerWithFixedPortsTestResource.class)
```

Comment:
If these tests are using Docker, they should only be enabled when `-Dtest-containers`.